src/System.Net.Http/src/System/Net/Http/HttpClient.cs

```diff
@@ -614,7 +614,7 @@ private static void CheckBaseAddress(Uri baseAddress, string parameterName)
             TaskCompletionSource<HttpResponseMessage> tcs)
         {
             LogSendError(request, cancellationTokenSource, "SendAsync", null);
-            tcs.TrySetCanceled();
+            tcs.TrySetCanceled(cancellationTokenSource.Token);
```

Comment:
@josguil Is this behavior different from Desktop or was this something that we've missed in the port?